metaflow/plugins/kfp/tests/.gitlab-ci.yml

```diff
@@ -17,8 +17,8 @@ variables:
   ARTIFACTORY_WEB_APP_URL: https://artifactory.zgtools.net/artifactory/webapp/#/packages/docker
   _SHARED_ENV_FILE: 'shared.env'
   _IMAGE_TAG_FILE: 'image_tag_file.txt'
-  DEPLOY_INTERNAL: "false"
-  DEPLOY_NONPROD: "true" # want to mimic the example projects, which run on nonprod
+  DEPLOY_INTERNAL: "true"
+  DEPLOY_NONPROD: "false" # don't run on nonprod until instability issues fixed
```

Comment:
best to be specific and say what instability issue exists, else one reading this will think it's due to this plugin, MF or KFP

Reply:
Good catch, we won't run on nonprod at all anymore, so I'll remove that comment entirely. I'll comment on why we run on internal instead.